app/webhost/encode_zip_to_base64.py

```diff
@@ -0,0 +1,13 @@
+import base64
```

Comment:
I am good with a py script, but if you prefer bash is also possible

`cat web.zip | base64 -w 0`